src/video_core/renderer_opengl/gl_shader_decompiler.cpp

```diff
@@ -79,6 +79,11 @@ const float fswzadd_modifiers_a[] = float[4](-1.0f,  1.0f, -1.0f,  0.0f );
 const float fswzadd_modifiers_b[] = float[4](-1.0f, -1.0f,  1.0f, -1.0f );
 )";
 
+enum HelperFunctions {
```

Comment:
```suggestion
enum class HelperFunction {
```

We generally avoid regular enums, as they pollute the surrounding scope and are also implicitly convertible. The following private helpers can also be introduced:

```cpp
void SetHelperEnabled(HelperFunction hf, bool enabled) {
    helper_functions_enabled[static_cast<u32>(hf)] = enabled;
}

bool IsHelperEnabled(HelperFunction hf) const {
    return helper_functions_enabled[static_cast<u32>(hf)];
}
```